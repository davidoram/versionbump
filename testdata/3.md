## 5.1.0
Stuff
## 4.9.0
Initial version