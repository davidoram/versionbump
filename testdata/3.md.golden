## 5.2.0

This text is added to the top of the file [ticket](http://link/to/a/ticket)

## 5.1.0
Stuff
## 4.9.0
Initial version